subt/doc/virtual-howto.md

```diff
@@ -0,0 +1,61 @@
+# SubT Challenge - Virtual Track HowTo
+
+Here are instructions how to build, run locally and deploy to AWS "robotika" solution docker.
+It is expected that you already have docker installed and your PC has compatible NVidia GPU.
+For official instructions how to start with Docker please look at:
+  https://bitbucket.org/osrf/subt/wiki/tutorials/SystemSetupDockerInstall
+
+
+## Prepare OSGAR build environment
+The convenience scripts for build and run are merged into OSGAR:
+```
+git clone https://github.com/robotika/osgar.git .
+cd ~/osgar/subt/docker
+```
+
+## Download official docker images
+The currently available images are listed at:
+  https://hub.docker.com/r/osrf/subt-virtual-testbed/tags
+Note, that you will need cloudsim_sim_latest, cloudsim_bridge_latest and subt_solution_latest
+```
+docker pull osrf/subt-virtual-testbed:cloudsim_sim_latest
+docker pull osrf/subt-virtual-testbed:cloudsim_bridge_latest
+docker pull osrf/subt-virtual-testbed:subt_solution_latest
+```
+
+## Build and run locally simple unittest image
+```
+cd ~/osgar/subt/docker
+./build.bash unittest
```

Comment:
I get lots of compilation followed with:
```
Install space: /home/developer/subt_solution/install
####
#### Running command: "make cmake_check_build_system" in "/home/developer/subt_solution/build"
####
####
#### Running command: "make install -j8 -l8" in "/home/developer/subt_solution/build"
####
Removing intermediate container 5b389879e713
 ---> a87b074c6553
Step 5/5 : ENTRYPOINT ["./run_solution.bash"]
 ---> Running in f05172a37940
Removing intermediate container f05172a37940
 ---> 46cb3a7db73f
[Warning] One or more build-args [user_id] were not consumed
Successfully built 46cb3a7db73f
Successfully tagged unittest:2019_Dec_06_2102
Error parsing reference: "unittest:" is not a valid repository/tag: invalid reference format
Built unittest:2019_Dec_06_2102 and tagged as unittest:latest and unittest:
```

The parsing error doesn't sound right.

Reply:
These two scripts are copy of original Ignition utilities. The error is misleading and we should probably tweak the scripts to our needs. I.e. it is expected at the moment.